check_logs.py

```diff
@@ -182,8 +178,9 @@ def boot_test(build):
     build = get_build()
     print(json.dumps(build, indent=4))
     print_yellow("Register clang error/warning problem matchers")
-    print("::add-matcher::.github/problem-matchers/compiler-source.json")
-    print("::add-matcher::.github/problem-matchers/compiler-non-source.json")
+    problem_matchers = glob.glob(".github/problem-matchers/*.json")
+    for problem_matcher in problem_matchers:
```

Comment:
Oooh yeah, I like this a lot. :)

You could lose the "problem_matchers" intermediate variable if you wanted to, and just put the glob on the `for` line.

Reply:
Good call, I did that initially then I think I was going to do something with `problem_matchers` then never did.